python/taichi/lang/transformer.py

```diff
@@ -922,3 +929,13 @@ def visit_Return(self, node):
                 ' Consider using a local variable to walk around.')
 
         return node
+
+
+class HackFunctionCall(ASTTransformerBase):
```

Comment:
Give this a more meaningful name, e.g. `TransformFunctionCallAsStmt`? Also explain that the purpose of this pass is to transform a standalone taichi function call expression into a statement?